### PR TITLE
Add logging to mobile view

### PR DESCRIPTION
**Pull request by LBHCallumM**
## Summary of Changes

Logs what operatives can see on the mobile view.

### CODE REVIEW

File: src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js [hunk 1]
```diff
@@ -54,28 +84,29 @@ const MobileWorkingWorkOrdersView = ({ currentUser }) => {
...
-              return ''
-            }
-          })()}
-          currentUser={currentUser}
-        />
-      ))
-    )
+    if (workOrders.length === 0) {
+      return <></>
+    }
```
Comment (LBHCallumM) [inline src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js:89]: Previous logic displayed '0' when empty

File: src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js [hunk 2]
```diff
@@ -9,11 +9,14 @@ import WarningInfoBox from '../../Template/WarningInfoBox'
 import Meta from '../../Meta'
 import { WorkOrder } from '../../../models/workOrder'
 
-const MobileWorkingWorkOrdersView = ({ currentUser }) => {
+const MobileWorkingWorkOrdersView = ({
+  currentUser,
+  loggingEnabled = true,
```
Comment (adamtry) [inline src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js:14]: Does this need a feature toggle?

Comment (LBHCallumM) [reply src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js:14]: its not a feature toggle

File: src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js [hunk 3]
```diff
@@ -99,6 +132,32 @@ const MobileWorkingWorkOrdersView = ({ currentUser }) => {
       .slice(0, 1)
   }
 
+  const logWorkOrders = async (
```
Comment (adamtry) [inline src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js:135]: This is good info to have and definitely very useful as-is. Could consider adding some timestamps so we won't need to look them up after?

So in progress work orders, started work orders, and current work orders as objects including id, start /end times, and scheduled start / end times.

But if this needs API changes then don't worry about it for now I guess - this is very good as-is

File: src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js [hunk 4]
```diff
@@ -99,6 +132,32 @@ const MobileWorkingWorkOrdersView = ({ currentUser }) => {
...
+    }
+
+    try {
+      await frontEndApiRequest({
+        method: 'post',
+        path: '/api/frontend-logging/operative-mobile-view-work-orders',
+        requestData: body,
+      })
+    } catch (error) {
+      // fail silently - user doesnt need to know if logging fails
```
Comment (adamtry) [inline src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js:157]: Could consider a console.error here just so we can investigate in the unlikely event of a failure

**Review by adamtry — APPROVED** [review]

File: src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js (hunk 3 above)
Comment (LBHCallumM) [reply src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js:135]: timestamps are set within the api

Comment (LBHCallumM) [reply src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js:135]: We could just log when returning data from the api. But I think whats more useful is knowing what the frontend sees. So we can see the result of any filtering logic on the frontend

Comment (LBHCallumM) [reply src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js:135]: If we find this useful, there is no reason we couldnt change the implementation

**Pull request merged**